pom.xml

```diff
@@ -24,17 +24,17 @@
   </parent>
   <groupId>org.apache.accumulo</groupId>
   <artifactId>accumulo-examples</artifactId>
-  <version>2.0.0-SNAPSHOT</version>
+  <version>2.1.0-SNAPSHOT</version>
   <name>Apache Accumulo Examples</name>
   <description>Example code and corresponding documentation for using Apache Accumulo</description>
   <properties>
-    <accumulo.version>2.0.0</accumulo.version>
+    <accumulo.version>2.1.0-SNAPSHOT</accumulo.version>
     <eclipseFormatterStyle>contrib/Eclipse-Accumulo-Codestyle.xml</eclipseFormatterStyle>
-    <hadoop.version>3.2.1</hadoop.version>
+    <hadoop.version>3.3.0</hadoop.version>
     <log4j.version>2.14.0</log4j.version>
-    <maven.compiler.release>8</maven.compiler.release>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.release>11</maven.compiler.release>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
     <zookeeper.version>3.4.14</zookeeper.version>
```

Comment:
This probably would benefit from a bump in ZK to match Accumulo, also.

Reply:
That is the next step. I wanted to get this in there first since the last time I attempted to bump ZK I ran into some issue with the change from ZK 3.4 to 3.5. Once I merge these changes I will work on that.

Reply:
Okay. I think ZK 3.4 has some issues with Java 11, but can't remember the details. There might be some test failures until that is updated.